collect_app/src/main/java/org/odk/collect/android/widgets/RangeWidget.java

```diff
@@ -179,6 +179,19 @@ private void setUpSeekBar() {
         seekBar.setMax(elementCount);
         seekBar.setProgress(progress);
         seekBar.setOnSeekBarChangeListener(this);
+        // fix Problems with Range Widgets with RTL languages #1983.
```

Comment:
This comment is not needed, you are in RangeWidget class so it's clear you solve the problem wit RangeWidget and `isRTL()` method name is also descriptive enough.